controllers/pitch.js

```diff
@@ -30,7 +35,23 @@ module.exports = {
       });
   },
   downVote: function (req, res) {
-    Pitch.updateOne({ _id: req.params.projectId }, { $inc: { votes: -1 } })
+    Pitch.updateOne(
+      { _id: req.params.projectId },
+      { $inc: { votes: -1 } }
+    )
+      .then(function (doc) {
+        res.json(doc);
+      })
+      .catch(function (err) {
+        res.json(err);
+      });
+  },
+  recordVotedProject: function (req, res) {
```

Comment:
I realized we can just require the User model through this controller and use it here, instead of creating a who new controller. This is the query that takes in the userId and projectId and pushes the projectId into the votedProjects array for the user. We can then check this array via the front end.